### PR TITLE
 Adding SSHKEY Repository as a parameter to scale-ci-baseline

### PR DESCRIPTION
**Pull request by piyushgupta1551**
Non-breaking change to parameterize the SSKHEY repo with redhat-perf as default as suggested.

### CODE REVIEW

**Comment by bpradipt** [comment]
@piyushgupta1551 do you want to add a comment in the commit "Fixes issue #72 " to reference the related issue

File: jjb/dynamic/scale-ci_baseline.yml [hunk 1]
```diff
@@ -137,6 +140,19 @@
         description: 'Duration to run'
         name: BASELINE_WORKLOAD_DURATION
         trim: 'false'
+    - string:
+        default: "github.com/redhat-performance/perf-dept.git"
+        description: 'Repository container ssh keys for Ansible and pbench server'
```
Comment (jtaleric) [inline jjb/dynamic/scale-ci_baseline.yml:145]: s/Repository container ssh keys for Ansible and pbench server/Repository for ssh keys/g

Comment (piyushgupta1551) [reply jjb/dynamic/scale-ci_baseline.yml]: @jtaleric , the description has been updated.

File: jjb/dynamic/scale-ci_baseline.yml [hunk 2]
```diff
@@ -14,12 +14,15 @@
...
         # Re-enable logging
         set -x
-        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
-        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
-        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
-        export PBENCH_SSH_PUBLIC_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf.pub
-        export PBENCH_SSH_PRIVATE_KEY_FILE=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf
-        chmod 600 ${PRIVATE_KEY}
+        git clone https://${SSHKEY_TOKEN}@${SSHKEY_REPO}
+        if [ -f ${WORKSPACE}/${SSHKEY_REPOPATH_PUB} ] && [ -f ${WORKSPACE}/${SSHKEY_REPOPATH_PRIV} ]
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_baseline.yml:18]: We need to update other jobs to do the same as they use the same logic to fetch the ssh keys.

File: jjb/dynamic/scale-ci_baseline.yml [hunk 3]
```diff
@@ -137,6 +140,19 @@
         description: 'Duration to run'
         name: BASELINE_WORKLOAD_DURATION
         trim: 'false'
+    - string:
```
Comment (chaitanyaenr) [inline jjb/dynamic/scale-ci_baseline.yml:143]: This change will work when we are calling the individual job, the following minor changes are needed to get it working with the pipeline: the pipeline/build scripts read the properties file/config with the parameters and call the jobs. so the sample properties file ( https://github.com/openshift-scale/scale-ci-pipeline/blob/master/properties-files/baseline.properties ) and build script needs to be updated with the new parameters ( https://github.com/openshift-scale/scale-ci-pipeline/blob/master/pipeline-scripts/baseline.groovy#L46 ).

File: jjb/dynamic/scale-ci_baseline.yml (hunk 2 above)
Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci_baseline.yml:18]: This can be tackled in a separate PR I guess.

File: jjb/dynamic/scale-ci_baseline.yml (hunk 3 above)
Comment (piyushgupta1551) [reply jjb/dynamic/scale-ci_baseline.yml:143]: makes sense @chaitanyaenr . I was looking at updating the individual tests as I'm not wanting to run them all. Would it be okay if I update the yaml, properties and groovy files for each test at a time and raise a pull request?

Comment (chaitanyaenr) [reply jjb/dynamic/scale-ci_baseline.yml:143]: Sounds good. Thanks.

Comment (piyushgupta1551) [reply jjb/dynamic/scale-ci_baseline.yml:143]: @chaitanyaenr - could you please review the most recent changes in this request?

**Pull request merged**